src/index.html

```diff
@@ -11,7 +11,186 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Name:
+            <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            How old are You?
+            <input type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100">
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            Full date of birth:
+            <input
+            type="date"
+            name="birthdate"
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the agreement:
+            <input
+            type="checkbox"
+            name="terms"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registrarion</legend>
+
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input
+            type="email"
+            name="email"
+            autocomplete="off"
+            placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Password:
+            <input
+            type="password"
+            minlength="8"
+            maxlength="32"
+            name="password"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input
+            type="radio"
+            name="love-cats"
+            value="y"
+            checked
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="love-dogs"
+            value="n"
+            >
+            No
+          </label>
```

Comment:
add one empty line between same level elements


```suggestion
          <label>
            Do you love cats?
            <input
            type="radio"
            name="love-cats"
            value="y"
            checked
            >
            Yes
          </label>
          
          <label>
            <input
            type="radio"
            name="love-dogs"
            value="n"
            >
            No
          </label>
```